setup.cfg

```diff
@@ -28,7 +28,7 @@ packages = find:
 setup_requires = setuptools_scm
 install_requires =
     numba
-    numpy<=1.22  # pinned for compatibility with numba
+    numpy<1.25
```

Comment:
Can we remove this pin? This seems like something that the solvers should handle, not us.

Reply:
Done, I also removed the `networkx` dependency.
We plan to support creating napari graphs from it, but at the moment, the package is unused.